cli/azd/pkg/project/service.go

```diff
@@ -105,3 +99,13 @@ func GetServiceResourceName(ctx context.Context, resourceGroupName string, servi
 
 	return res[0].Name, nil
 }
+
+// GetServiceResources gets the resources tagged for a given service
+func GetServiceResources(ctx context.Context, resourceGroupName string, serviceName string, env *environment.Environment) ([]azcli.AzCliResource, error) {
+	azCli := azcli.GetAzCli(ctx)
+	query := fmt.Sprintf("[?tags.\"azd-service-name\" =='%s']", serviceName)
+
+	return azCli.ListResourceGroupResources(ctx, env.GetSubscriptionId(), resourceGroupName, &azcli.ListResourceGroupResourcesOptions{
+		JmesPathQuery: &query,
+	})
+}
```

Comment:
What was the reason for breaking this out and requiring both `GetServiceResources` and `GetServiceResourceName` ? Do we have a scenario where we expect multiple resources to get tagged with the `azd-service-name` tag?

Reply:
> Do we have a scenario where we expect multiple resources to get tagged with the azd-service-name tag?

Per the discussion above, I think we are going to end up in that place long term, since you may want to deploy the same service code to two different resources (which are in different regions).

The other thing pushing me towards doing this was in the case where `provision` had not yet run,  `GetServiceResourceName` would return a value based on a convention we used to use which was not going to be correct.